Docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: "3"
+
+services:
+  Development:
+    env_file:
+        - .env
+    build: . 
```

Comment:
You need to specify which stage you want to build. If you don't, it'll build the lowest one in the Dockerfile.

You need to expand this build section:
```
    build: 
      context: .
      target: development
```